server/utils/organisationUtils.ts

```diff
@@ -10,7 +10,7 @@ import type { Prison } from '@prison-register-api'
 
 const organisationFromPrison = (organisationId: Organisation['id'], prison: Prison): Organisation => {
   const { addressLine1, addressLine2, town, county, postcode, country } = prison.addresses[0]
-  const categories = prison.categories.length > 0 ? prison.categories.sort().join('/') : 'Not available'
+  const categories = prison.categories.length > 0 ? prison.categories.sort().join('/') : ''
```

Comment:
Could this be simplified? ie
```
const categories = prison.categories.sort().join('/')
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join#return_value

Reply:
oops, of course it can... thanks for spotting